_includes/description.html

```diff
@@ -1,6 +1,11 @@
 <h2>Meet the Pod</h2>
 <div class="card-container">
     {% for item in site.data.fellows %}
+    {% if item.page-name %}
+    <a href="/description/{{ item.page-name }}">
```

Comment:
The `</a>` is missing as well as the link text. I assume this would also be `{{ item.page-name}}`?